### ISSUE TITLE
Лаги при работе с графиком

### ISSUE THREAD

**Issue by funownz** [SBProX/SBProX-Dev#1562]
Работаю с графиком нефти Claster profile tf 1 min и 5 min. Если работать с близкой датой, например с прошлой недели, то всё абсолютно нормально. Если же брать дату например 31.01.2019, то при прокрутке графика вправо влево всё жутко тормозит.

Присутствую два индикатора Cloud. на график выведен период в ОДИН день.

С одними  и теми же настройками близкие даты не тромозят, а даты 2 года назад тормозят.

**Comment by SBProX** [comment]
Да, тормозят, потому что слишком много данных для таких тайм фреймов.
Мы не рекомендуем включать на минутных графиках два года истории, этот как минимум бессмысленно.

**Comment by funownz** [comment]
Вы меня не правильно поняли. Загружен был только ОДИН день. И он тормозил.
При загрузке такого же дня в недавней истории тормозов не было. Я не грузил
два года. Я грузил ОДИН день.

пн, 5 окт. 2020 г., 21:23 SBPro X <notifications@github.com>:

**Comment by funownz** [comment]
> Да, тормозят, потому что слишком много данных для таких тайм фреймов.
> Мы не рекомендуем включать на минутных графиках два года истории, этот как минимум бессмысленно.

А смысл простой, я открываю по одному дню на истории для исследования поведения моей стратегии на истории. Повторюсь, я не гружу два года на графике. Я гружу один! день. Возможно при подгрузке у вас данные только показываются за указанный в верху угла интервал, а гразятся и тормозят за весь срок, тогда это проблема...
